lib/bus.js

```diff
@@ -683,18 +683,27 @@ module.exports = function bus(conn, opts) {
                             let propAccessMode = Object.keys(propObj)[0]
                             let propType = propObj[propAccessMode]
 
+                            //get user-defined getter/setters
+                            let propGetter = obj[ifaceName].__lookupGetter__(propName)
+                            let propSetter = obj[ifaceName].__lookupSetter__(propName)
+
+                            //use default getter/setter if not user-defined
+                            if (!propGetter) {
```

Comment:
Can you make the check more explicit, like `if (propGetter == null)` rather than coercing the value to a boolean (we said we'd try to avoid coercing values to boolean as much as possible)